main.go

```diff
@@ -122,6 +157,26 @@ func main() {
 					},
 				},
 			},
+			{
+				Name:     "renew-cert",
+				HelpName: "renew-cert <EST Server>",
+				Usage:    "Renew device's TLS keypair used with device-gateway",
+				Action: func(c *cli.Context) error {
+					return renewCert(c)
+				},
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:  "pkcs11-key-ids",
+						Value: "01,07",
```

Comment:
I suppose token labels of these slots should match the p11.label param, otherwise aktualizr won't find them?